i18n/zh-CN/docusaurus-plugin-content-blog/2024-03-04-kcl-0.8.0-release/index.md

```diff
@@ -0,0 +1,466 @@
+---
+slug: 2024-03-04-kcl-0.8.0-release
+title: KCL v0.8.0 重磅发布 - 融汇社区生态
+authors:
+  name: KCL Team
+  title: KCL Team
+tags: [Release Blog, KCL]
+---
+
+## 简介
+
+KCL 团队很高兴地宣布 **KCL v0.8.0 新版本现在已经可用**！本次发布为大家带来了三方面的重点更新：**语言**、**工具链**、**社区集成 & 扩展支持**。
+
+- _使用功能更完善错误更少的 KCL 语言、工具链和 IDE 提升代码编写体验和效率_
+- _更加全面丰富的社区生态集成，改善运维体验_
+- _更加丰富的 KCL 三方库模型，更加轻松的与云原生生态集成_
+
+进一步您可以在 [KCL v0.8.0 发布页面](https://github.com/kcl-lang/kcl/releases/tag/v0.8.0) 或者 [KCL 官方网站](https://kcl-lang.io) 获得下载安装指南和详细发布信息。
+
+[KCL](https://github.com/kcl-lang/kcl) 是一个 CNCF 基金会托管的面向云原生领域开源的基于约束的记录及函数编程语言，期望通过成熟的编程语言技术和实践来改进对大量繁杂配置比如云原生 Kubernetes 配置场景的编写，致力于围绕配置的模块化、扩展性和稳定性，打造更简单的逻辑编写体验，构建更简单的自动化和生态集成路径。
+
+## 语言更新
+
+### 🚗 语法语义更新
```

Comment:
Consider all commits in v0.7.5-0.8.0, especially the issues raised by users and potential behavioral changes that may break.

Reply:
fixed